nannou_egui_demo_app/src/main.rs

```diff
@@ -20,10 +22,11 @@ fn model(app: &App) -> Model {
         .unwrap();
     let window = app.window(w_id).unwrap();
     let mut egui = Egui::from_window(&window);
-    let mut egui_demo_app = egui_demo_lib::WrapApp::default();
-    let proxy = app.create_proxy();
-    egui.do_frame_with_epi_frame(proxy, |ctx, epi_frame| {
-        egui_demo_app.setup(&ctx, epi_frame, None);
+    let mut egui_demo_app = egui_demo_lib::DemoWindows::default();
```

Comment:
Not sure the best way to clean this up. Maybe make the `Egui` constructors accept an `App` instance so we can wire that up internally?